build.yml

```diff
@@ -307,6 +307,29 @@ docker-targets:
       xorg-x11-fonts-Type1
       zlib
 
+  lambdaamazonlinux2:
```

Comment:
I'd prefer calling this `amazonlinux2_lambda`

Reply:
I agree. Looks more consistent in `build list-targets` output.